### PR TITLE
Suggested changes to Expected Behavior section

### PR DESCRIPTION
**Pull request by danmactough**
This is only a first pass. As noted in #6, betted alignment with forthcoming Core Values rewrite will needed.

### CODE REVIEW

**Comment by cpolitano** [comment]
:+1:

**Comment by kylehill** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -22,17 +22,17 @@ We want our team to be a fun, productive, and safe space for all members. In add
...
+**Be collaborative.** Involve your colleagues -- senior and junior -- in brainstorms, sketching sessions, code reviews, planning documents, and the like. It’s not only okay to ask for help or feedback often, it’s unacceptable not to do so. Don’t succumb to either impostor syndrome (believing that you don’t deserve to be here) or [blowhard syndrome](http://xuhulk.tumblr.com/post/110549967516/stop-blowhard-syndrome) (believing you can do no wrong). Recognize that in addition to asking for feedback, you are similarly obligated to give it.
 
-**Be generous and kind in both giving and accepting critique.** Critique is a natural and important part of our culture. Good critiques are kind, respectful, clear, and constructive, focused on goals and requirements rather than personal preferences. You are expected to give and receive criticism with grace.
+**Be generous and kind in both giving and accepting critique.** Critique is a natural and important part of our culture. Good critiques are kind, respectful, clear, and constructive, and focused on goals and requirements rather than personal preferences. You are expected to give and receive criticism with grace.
```
Comment (ecgeiser) [inline README.md]: would be helpful to provide examples or link to a well-written article about giving respectful criticism and accepting it with grace.  will add if I find anything

File: README.md [hunk 2]
```diff
@@ -22,17 +22,17 @@ We want our team to be a fun, productive, and safe space for all members. In add
...
 
-**Be humane.** Be polite and friendly in all forms of communication, especially remote communication, where opportunities for misunderstanding are greater. Use sarcasm carefully. Tone is hard to decipher online; make judicious use of emoji to aid in communication. Use video hangouts and IRL meetings when it makes sense; face-to-face discussion benefits from all kinds of social cues that may go missing in other forms of communication.
+**Be humane.** Be polite and friendly in all forms of communication, especially remote communication, such as Slack and email, where opportunities for misunderstanding are greater. Use sarcasm carefully. Tone is hard to decipher online; make thoughtful use of emoji to aid in communication. Use video hangouts and IRL meetings when it makes sense; face-to-face discussion benefits from all kinds of social cues that may go missing in other forms of communication.
```
Comment (ecgeiser) [inline README.md]: `make thoughtful use of emoji to aid in communication` :smile:

**Comment by gfogle** [comment]
this came up in conversation and one of the adds to this section "assume no ill will" would be beneficial. pairs nicely with this article: https://hbr.org/2015/12/a-mental-trick-to-help-with-challenging-conversations

**Comment by danmactough** [comment]
@gfogle That's a great article. How should we incorporate it?

File: README.md (hunk 1 above)
Comment (ecgeiser) [inline README.md]: This article popped up in the #random dctech slack channel and seemed somewhat relevant to our code of conduct discussions: https://hbr.org/2016/01/defusing-an-emotionally-charged-conversation-with-a-colleague

Not sure how/whether to incorporate it into our code of conduct, but it seems to fit with the idea of being kind/generous in our interactions

Comment (danmactough) [inline README.md]: Maybe in the [Further Reading](https://github.com/socialtables/code-of-conduct#further-reading) section?

Comment (cpolitano) [inline README.md]: @ecgeiser @danmactough I think this article is really, really great. The article that @gfogle posted is awesome, too. They are both so important that I suggest adding them as inline links, which gives them a lot more emphasis, rather then relegating them to the Further Reading, where they would be easy to skip over. 

For the article posted by Erica, we could add an additional sentence to the above paragraph that says "If a discussion becomes emotionally charged, try [using these techniques](https://hbr.org/2016/01/defusing-an-emotionally-charged-conversation-with-a-colleague) to defuse the situation and get the conversation back on track."

Comment (danmactough) [inline README.md]: @cpolitano Would you mind updating this PR to incorporate these comments?

**Pull request merged**